assets/resourceSnippets/Automanage Best Practices DevTest Assignment.snippet.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "prefix": "arm-automanage-best-practices-devtest-assignment",
```

Comment:
The prefix is generally kept relatively short.  Users can still see and search on the description.
```suggestion
        "prefix": "arm-automanage-bp-devtest-assignment",
```
<img width="933" alt="image" src="https://user-images.githubusercontent.com/6913354/189996853-86e65d2c-9765-46a7-9e48-c4b0c1ee8ed2.png">

Reply:
done